test/acceptance.sh

```diff
@@ -223,4 +224,39 @@ if [[ "$(command -v shellcheck)" ]]; then
   fi
 fi
 
+# poor man's style guide checking
+echo "==> b3bp style guide"
+pushd "${__root}" > /dev/null
+
+failed="false"
+
+while IFS=$'\n' read -r file; do
+  [[ "${file}" =~ ^\./node_modules/ ]] && continue
+  [[ "${file}" =~ ^\./website/\.lanyon/ ]] && continue
+
+  echo -n "    ${file}.. "
+
+  if egrep -Hv '^ *#' "${file}" \
+    | grep -vF "\$'" \
+    | grep -vF '$"' \
+    | grep -vF '\$' \
+    | egrep --color=always '\$[^\{\(]' \
+    >> "${__accptstTmpDir}/style.err"; then
+      echo "✗"
+      failed="true"
+      continue
+  fi
+
+  echo "✓"
+done <<< "$(find . -type f -iname '*.sh')"
+
+popd > /dev/null
+
+if [[ "${failed}" = "true" ]]; then
+  echo
+  cat "${__accptstTmpDir}/style.err"
+  echo
+  exit 0
```

Comment:
Also, previous test failure will call `exit n` with `n > 0` so the exit 0 isn't an issue here.